src/tonstakers.ts

```diff
@@ -414,6 +422,50 @@ class Tonstakers extends EventTarget {
     }
   }
 
+  async activeWithdrawalNFTs(): Promise<NftItemWithEstimates[]> {
+    try {
+      const poolData = await this.fetchStakingPoolInfo();
+      const withdrawalPayout = poolData.poolFullData.withdrawal_payout;
+      return await this.getFilteredByUserNFTs(withdrawalPayout)
+    } catch (error) {
+      console.error(
+        "Failed to get active withdrawals:",
+        error instanceof Error ? error.message : error,
+      );
+      throw new Error("Failed to get active withdrawals.");
+    }
+  }
+
+  private async getFilteredByUserNFTs(payoutAddress: string): Promise<NftItemWithEstimates[]> {
```

Comment:
this method probably should be cached with key including the payout address

Reply:
done